XMLProjectIS/MetaData.cs

```diff
@@ -6,16 +6,15 @@
 //
 //    var welcome = Welcome.FromJson(jsonString);
 
-namespace QuickType
```

Comment:
Namespace of CS file changed from "QuickType" to Movies which is relevant to the Project

Reply:
Yes, good.